packages/android_alarm_manager_plus/example/android/app/build.gradle

```diff
@@ -35,6 +35,10 @@ android {
         targetCompatibility JavaVersion.VERSION_17
     }
 
+    kotlinOptions {
+        jvmTarget = 17
```

Comment:
Had to do, so Android Studio doesn't pick 21 which is not supported yet but used Kotlin versions.
In fact, moving to Java 17 is something I want to do to the rest of plugins to be up to date. But such change will happen in another set of PRs.